src/client/components/ProductDetail/ProductDetail.component.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './ProductDetail.styles.css';
+import Button from '../Button/Button.component';
+
+export default function ProductDetail({
+  product,
+  quantityCount,
+  setQuantityCount,
+}) {
+  function decreamentQuantityCount() {
+    setQuantityCount((prevCount) => prevCount - 1);
+  }
+
+  function increamentQuantityCount() {
+    setQuantityCount((prevCount) => prevCount + 1);
+  }
+
+  function handleQuantityCountChange(e) {
+    e.preventDefault();
+    setQuantityCount(e.target.value);
+  }
+
+  const { name, price, picture } = product;
+  return (
+    <div className="productDetail">
+      <article className="productDetail-container">
+        <section className="image-container">
+          <img src={picture} alt={name} />
+        </section>
+        <section className="info-container">
+          <h3>{name}</h3>
+          <div className="info-container-stripes">
+            <div className="info-container-stripe">
+              <p>2lt pot </p>
+              <p>DKK &nbsp;{price}</p>
+            </div>
+            <div className="info-container-stripe">
+              <p>Quantity: &nbsp;</p>
```

Comment:
I'd try to move away from having so many nbsp in the markup.
You likely want to be using CSS `margin` instead!

Reply:
Would love to keep the first one since we are talking about a price here.  I found the use of &nbsp; in this case to be appropriate, but of course, can remove it anytime.
Removed the second one from Quantity: &nbsp; added a margin-left instead.